tests/test_h1_client.c

```diff
@@ -2722,7 +2722,9 @@ static int s_error_from_outgoing_body_read(struct aws_input_stream *body, struct
     (void)dest;
 
     struct error_from_callback_tester *error_tester = body->impl;
-    ASSERT_SUCCESS(s_error_from_callback_common(error_tester, REQUEST_CALLBACK_OUTGOING_BODY));
+    if (s_error_from_callback_common(error_tester, REQUEST_CALLBACK_OUTGOING_BODY)) {
```

Comment:
This is a part of a test where we purposefully return error codes at various callbacks.
These macros print stuff out when they fail, but it's misleading in this case because the test is **supposed** to return an error here.

I had a failure in this test and the first thing I chased was the output from this macro, but it was a [red herring](https://en.wikipedia.org/wiki/Red_herring).